src/transforms/babel.js

```diff
@@ -239,7 +239,7 @@ async function getEnvConfig(asset, isSourceModule) {
 
 const envCache = new Map();
 
-async function getEnvPlugins(targets) {
+async function getEnvPlugins(targets, useBuiltIns) {
```

Comment:
According to the [babel preset-env doc](https://babeljs.io/docs/plugins/preset-env/#usebuiltins), the default value of `useBuiltIns` is `false`, so what about make the second parameter of `getEnvPlugins` with a default `false`, like
```js
async function getEnvPlugins(targets, useBuiltIns = false) { ... }
```

Reply:
There does seem to be a little precedent to use a default argument, and it does seem to make usage a bit clearer. This does not actually change any functionality, since the argument is used for a truthy check, and undefined is falsy.